app/scripts/utils/reduce.js

```diff
@@ -1,9 +1,9 @@
 /**
- * Pure functional reducer. Can be used for currying stuff. Check out
- *   `flatten.js`.
+ * Pure functional reducer. Can be used for currying stuff. Check out `flatten.js`.
  * @param   {function}  f  Reducer function.
  * @return  {array}  Curried function that accepts an array to be reduced.
  */
 const reduce = (f) => (x) => Array.prototype.reduce.call(x, f);
 
+// TODO(Trevor): Not referenced anywhere. Remove?
```

Comment:
This and `flatten` are incorrectly implemented and not referenced anywhere. I'm thinking we can remove.